### ISSUE TITLE
zoom in to muni routes

### ISSUE THREAD

**Issue by lesliehoneybeefield** [ljuba/Transporter-iPhone#13]
allow users of the iphone app to "zoom in" and explore details of the transit routes.

**Comment by rodericj** [comment]
This is actually done if I'm not mistaken. Solved by #34

**Comment by lesliehoneybeefield** [comment]
How does the user do this? As a user, I want to be able to zoom in on the map after selecting the line from the Lines tab. I just downloaded the July 5 update, and I do not have this functionality -- plus the N lines on muni is causing the app to crash.

**Comment by rodericj** [comment]
This is fixed in the latest from master. The fix was most definitely after the July 5th App store build.

If you are able to build and run the app on your device with XCode and your paid Apple developer account, you should be able to see the fix when you run the app. If not, you'll have to wait until Apple accepts the newest submission to the app store. Unfortunately, there is no ETA on this. If I had to guess I'd say a week. But it's entirely in Apple's control right now.

**Comment by lesliehoneybeefield** [comment]
Did you see the same crashes I am seeing when selecting the N line? I am also seeing it with J, but not M. Haven't tested extensively.

**Comment by rodericj** [comment]
Yes, those are a separate issue, but they should be resolved with this new fix. @ljuba knows a bit more about the crashes than I.

Basically, the entire first screen in the lines tab is now totally re-worked. It should look the same but you'll notice that the crashes should be gone and also the map should allow user interaction and the map should be less grainy.
